backend/server.js

```diff
@@ -21,6 +23,134 @@ app.get("/", (req, res) => {
   res.send("Hello Technigo!");
```

Comment:
Could add the endpoints here with the npm package "express-list-endpoints" if you wanted it might be easier to refer back to the project that way. Alternatively you could add them to your ReadMe file